KarhooUISDK/Screens/QuoteList/QuoteListViewController.swift

```diff
@@ -39,7 +39,7 @@ final class KarhooQuoteListViewController: UIViewController, BaseViewController,
             addressPickerView,
             quoteCategoryBarView,
             quoteSortView,
-            legalDisclaimerLabel
+            legalDisclaimerContainer
```

Comment:
This view should be hidden instead of legalDisclaimerLabel in setHeaderDisabled/setHeaderEnabled, in other case empty view will stay on stack.

Reply:
You are right, of course. Fixed.